### DISCUSSION TITLE
How do I make link note only use links in one section?

### DISCUSSION THREAD

**Discussion by laurastephsmith** [SkepticMystic/breadcrumbs#363] [Q&A]
I basically want a link note, but only want links under a particular heading to be used. Without an option for that in a link note, I'm trying to do with with a dataview note. Here's a minimal sample note so you can see what I'm trying to do:

```
---
BC-dataview-note: 'SOME QUERY I HAVE NOT WORKED OUT YET'
BC-dataview-note-field: child taxon
---

# Daffodils
 
## Other section with links I want ignored
- [[Daisy]]
- [[Sunflower]]

## Subspecies
- [[Narcissus pseudonarcissus subsp. pseudonarcissus (Wild Daffodil)]]
```
What I want is the links under the "Subspecies" heading to be children of the current note. The first problem I've run into is '[[]]' and '[[#]]' aren't valid syntax here. How do I refer to the current note? And then can I use filter() to somehow ignore the links in the other section?

Thanks for any help! L

**Comment by michaelpporter** [comment]
Shipped in v4: set `BC-list-note-section: "<heading>"` in a list note's frontmatter to scope the list-note builder to just that heading's section (up to the next equal/higher heading or end of file). Closing as resolved — thanks!
